### ISSUE TITLE
HackLA Project Page Revision 

### ISSUE THREAD

**Issue by BryonPm** [hackla-engage/start-here#21]
### Overview
We need to revise our project page with updated content.  

### Action Items

- [x] Add photo/illustration for project background image.
- [x] Replace project description content with one sheet referenced content.
- [x] Update meeting time to include Sunday 6pm-8pm CST.
- [x] Add Product Management under "Open Roles".
- [ ] Update Wiki photo under "Getting Started". 

### Resources/Instructions
[Current Project Page](https://www.hackforla.org/projects/engage)

**Comment by BryonPm** [comment]
@ExperimentsInHonesty I checked the HackforLA and Product Management folders for a guide or basic instruction on how to update team pages. I found a document, but it looks like it was unfinished. Please advise.
Blocks-Unfinished guide on appropriate team page updating
ETA: Unknown
Availability-Monday 6-9

**Comment by BryonPm** [comment]
Progress: Issue was escalated to Bonnie on 4/18 for completion.
Blocks: Development Team Backlog
ETA: Unknown 
Avaliability: Mon-Thur 6-10 CST

**Comment by BryonPm** [comment]
**Progress:** All content was approved and sent to HackforLA website team (Glen Florendo). Edited Hero image was received by Glen on 5/29. 
**Blocks:** Development Team Backlog 
**ETA:** Unknown
**Availability:** Mon-Thur 7-9 CST

**Comment by BryonPm** [comment]
Progress: Team page on the HackforLA website was updated with new Hero image and content. There was an issue with Hero image clarity and size. Issue will be re-opened and website team will make changes. 
Blocks:Glen Florendo needs to check with another PM for next steps on image.
ETA: Unknown
Availability: Mon-Thur 7-9 CST

**Comment by BryonPm** [comment]
Progress: Glen Paul Florendo on the website team reached out with recommendations regarding Hero image. Please see screenshot below for conversation.
Blocks: Team should agree on image quality and whitespace issue.
ETA: Unknown
Availability: Mon-Thur 7-9 CST

<img width="714" alt="Screen Shot 2021-06-30 at 6 58 48 PM" src="https://user-images.githubusercontent.com/76059921/124045491-5e66f800-d9d5-11eb-8d81-a264e49976c1.png">

**Comment by rmayott** [comment]
@BryonPm 
I increased the image size from 1500x700 (72dpi) to 4500x2100 (216dpi). Should solve the problem.
Link to the new image [here](https://drive.google.com/file/d/1P0m4tOu40c3hUfH6rNzoUsM6ubddYKzu/view?usp=sharing).

**Comment by BryonPm** [comment]
Progress: I reached out to Sihem through Slack to re-open or establish new issue for this Hero image error. 
Blocks: Response from HFLA Website Team PM
ETA: Unknown
Availability: Mon-Thur 7-9 CST

**Comment by ExperimentsInHonesty** [comment]
I opened up a new issue on the website repo https://github.com/hackforla/website/issues/1923

**Comment by BryonPm** [comment]
@ExperimentsInHonesty Please find the communication thread regarding the Hero image below
<img width="757" alt="Screen Shot 2021-07-11 at 6 31 14 PM" src="https://user-images.githubusercontent.com/76059921/125213017-7a408880-e276-11eb-973b-1220484c99e1.png">

**Comment by BryonPm** [comment]
Current status is here: https://github.com/hackforla/website/issues/1923

**Comment by ExperimentsInHonesty** [comment]
the hackforla.org page has been updated.
